### ISSUE TITLE
Typescript definition error

### ISSUE THREAD

**Issue by DerekLiang** [aaron9000/react-image-timeline#20]
## The error
Here is the error message when I used the react typescript template:

```
TypeScript error in <root>/src/index.tsx(7,22):
File '<root>/node_modules/react-image-timeline/index.d.ts' is not a module.  TS2306

     5 | import * as serviceWorker from './serviceWorker';
     6 | 
  >  7 | import Timeline from 'react-image-timeline';
```

## To reproduce it:
```bash
npx create-react-app my-app --template typescript
# add the following line to index.tsx file as shown in the error message
# import Timeline from 'react-image-timeline';
```

## The workaround 
If I replaced the `<root>/node_modules/react-image-timeline/index.d.ts` with `<root>/node_modules/react-image-timeline/dist/timline.d.ts`,  it worked as expected.

**Comment by DerekLiang** [comment]
It seems that the following seems to be working
```ts
export * from './dist/timeline';
import { default as d } from './dist/timeline';

export default d;
```

**Comment by aaron9000** [comment]
@DerekLiang thank you. I will look into this.

**Comment by aaron9000** [comment]
@DerekLiang - I just updated the library. It should be fixed in `3.2.6`
